benchmark/sparse/linear_regression/run_sparse_benchmark.py

```diff
@@ -3,38 +3,60 @@
 """
 
 import argparse
+import time
+import numpy as np
 
-import keras_tf_model
+import keras_sparse_model
 import mxnet as mx
 import mxnet_sparse_model
 from scipy import sparse
 
+from keras import backend as K
+from keras.utils.data_utils import prepare_sliced_sparse_data
 
-def invoke_benchmark(batch_size, epochs):
-    train_data = mx.test_utils.rand_ndarray(shape=(1000, 10), stype='csr')
-    train_label = 3 * train_data
-    eval_data = mx.test_utils.rand_ndarray(shape=(1000, 10), stype='csr')
-    eval_label = 3 * eval_data
-
-    print("Running Keras-TF sparse benchmark script")
-    keras_tf_model.run_benchmark(train_data=sparse.csr_matrix(train_data.asnumpy()),
-                                 train_label=sparse.csr_matrix(train_label.asnumpy()),
-                                 eval_data=sparse.csr_matrix(eval_data.asnumpy()),
-                                 eval_label=sparse.csr_matrix(eval_label.asnumpy()),
-                                 batch_size=int(batch_size),
-                                 epochs=int(epochs))
 
-    print("Running MXNet sparse benchmark script")
-    mxnet_sparse_model.run_benchmark(train_data=train_data, train_label=train_label, eval_data=eval_data,
-                                     eval_label=eval_label, batch_size=int(batch_size), epochs=int(epochs))
+def invoke_benchmark(batch_size, epochs):
+    # Fix seed for randomness
+    np.random.rand(7)
```

Comment:
can we remove this and run a few time (e.g. 5) and take an average of the speed.